### ISSUE TITLE
[Request] "Please choose the profile you want to use" prompt unnecessary when I only have one profile?

### ISSUE THREAD

**Issue by sw-yx** [aws-amplify/amplify-cli#4581]
**Is your feature request related to a problem? Please describe.**

in the typical `amplify init` workflow, we will have a lot of users that just have one profile (do we have data on what % of users have multiple profiles? would motivate this a lot)

currently our CLI just prompts when it isnt needed:

```bash
? Do you want to use an AWS profile? Yes
? Please choose the profile you want to use
> default
```

It's also unclear to me when I would ever NOT want to use an AWS profile - we don't explain this in our [tutorial](https://docs.amplify.aws/start/getting-started/setup/q/integration/react#initialize-a-new-backend) - so if we always say Yes, why do we ask it?

**Describe the solution you'd like**
Just want to take a closer look at this new user hot path so that we dont have more prompting than really necessary, for the 99% usecase.

**Comment by attilah** [comment]
@sw-yx thanks for the suggestion, that should be a default behavior for select lists in the whole CLI.

Please open an issue in the docs repo regarding what you miss in the given pages that should be there so we can make it better.

**Comment by renebrandel** [comment]
Addressed by #6949
